.github/workflows/test.yml

```diff
@@ -228,4 +227,20 @@ jobs:
       needs_zebra_state: true
       saves_to_disk: false
       disk_suffix: tip
-      zebra_state_path: '/var/cache'
+      root_state_path: '/var/cache'
+      zebra_state_dir: 'zebrad-cache'
+
+  lightwalletd-transactions-test:
+    needs: build
+    uses: ./.github/workflows/gcp-test-deploy.yml
+    if: ${{ github.event.inputs.regenerate-disks != 'true' || github.event.inputs.run-full-sync != 'true' }}
```

Comment:
Question: why is this condition different from all the other jobs?

(It's ok to make them all the same in this PR.)